src/codegeneration/FromOptionsTransformer.js

```diff
@@ -109,8 +111,11 @@ export class FromOptionsTransformer extends MultiTransformer {
     if (options.typeAssertions) {
       // Transforming member variabless to getters/setters only make
       // sense when the type assertions are enabled.
-      if (options.memberVariables) append(MemberVariableTransformer);
-      append(TypeAssertionTransformer);
+      if (transformOptions.memberVariables) append(MemberVariableTransformer);
+      this.append((tree) => {
```

Comment:
I think this was left over from my attempt to fix a problem that I later fixed another way. Done.